src/codemod.py

```diff
@@ -802,74 +737,111 @@ def _terminal_restore_color():
   import curses, sys
   sys.stdout.write(curses.tigetstr('sgr0'))
 
-def print_through_less(text):
-  """
-  Prints `text` to standard output.  If `text` wouldn't fit on one screen (as
-  measured by line count), make output scrollable a la `less`.
-  """
-  from tempfile import NamedTemporaryFile
-  tempfile = NamedTemporaryFile()
-  tempfile.write(text)
-  tempfile.flush()
-  os.system('less --no-init --quit-if-one-screen %s' % tempfile.name)
-
-
 #
 # Code to make this run as an executable from the command line.
 #
 
-class _UsageException(Exception): pass
-
 def _parse_command_line():
-  import getopt, sys, re
-  try:
-    opts, remaining_args = getopt.gnu_getopt(
-        sys.argv[1:], 'md:',
-        ['start=', 'end=', 'extensions=', 'exclude_paths=',
-         'include-extensionless', 'editor=', 'count', 'test'])
-  except getopt.error:
-    raise _UsageException()
-  opts = dict(opts)
-
-  if '--test' in opts:
+  import argparse, re, sys, textwrap
```

Comment:
Would it be better to import these at the top level? I think that's what this PEP suggests: https://www.python.org/dev/peps/pep-0008/#imports

Also, I know this isn't consistent within codemod itself, so I'm sorry to point it out here. But I do plan on making this file a little more PEP compliant, and it wouldn't hurt to start now! :speak_no_evil: